hubgrep/frontend_blueprint/templates/components/content_swap/_content_swap.scss

```diff
@@ -0,0 +1,19 @@
+.content_swap {
+  .expanded {
+    visibility: hidden;
+    position: absolute;
+  }
+
+  &:hover {
+    .default {
+      visibility: hidden;
+      position: absolute;
+    }
+
+    .expanded {
+      visibility: visible;
+      position: unset;
+    }
+  }
+}
+
```

Comment:
new general template macro for this expand/swap stuff - since its used in more than one place.